include/insns/RV32I.h

```diff
@@ -257,7 +257,7 @@ namespace SST{
           R->RV32_PC += Inst.instSize;
         }else{
           R->RV64[Inst.rd] = 0x00;
-          R->RV64[Inst.rd] = (Inst.imm << 12);
+          SEXT(R->RV64[Inst.rd], Inst.imm << 12, 32); 
```

Comment:
When running RV64, instruction lui	sp,0xffff8
		* Results in :0x00000000ffff8000
		* Correct result: 0xffffffffffff8000
		* Solution: needs SEXT on immd after shift